DIS.html

```diff
@@ -109,25 +92,56 @@ <h3>Section Three - Preparing your SD Card</h3>
     <ol>
         <li>Download the <a href="https://github.com/AuroraWright/Luma3DS/releases/latest"> latest Version of Luma3DS
             (Currently 9.0)</a></li>
-        <li>Download the <a href="https://github.com/zoogie/b9sTool/releases">latest Version of this fork of b9sTool</a>

```

Comment:
I do specifically NOT want to have any magnet links! Please use zoogie's b9sTool.
It has only one option, hence is less confusable and does not require the additional files.

Reply:
We specifically do not want to use zoogie's b9sTool due to having Nintendo's binaries hardcoded into it, and being directly hosted in GitHub.

Reply:
While i do understand that, I'd like to keep using zoogie's.
It is simpler for the user and in the end, the user acquires a copy of the binaries through this guide one way or another. There is no real need to hide that fact and therefor make it unnecessarily hard for the user.
Also also: having another set of files increases the chance of a file corruption affecting us, torrents tend to be fragile (at least in my past experience.)

Reply:
think we left out a warning by mistake that says they're magnet links 
`-If you cannot open Magnet links then you will need to download a torrent client. Deluge or Qbittorent is recommended`
This isn't even much more work than before. Most of us agree at some point we have to stop making it idiotproof.

Reply:
It being 'idiotproof' is the reason i wrote this guide and i would like to keep it that way.
I've had plenty of people that simply didnt know how to use torrents, were condused how to then use the magnet links and when the download actually finished - it is after all a step we can cut out.
In my mind, it is similar as to why we recommend DIS instead of explaining how to use TADPole.

Reply:
Lol at "We don't want to host Nintendo's firmware binaries in Zoogie's version". Hosting links to full Native Firms and game roms is ok, but having 30KB baked into b9sTool (3% of native firm) ... now that's a step too far!! XD
Absolutely silly double standard.

Reply:
I mean if you wanted us to, we could fully blacklist the tool from the server and be done with it. I'd rather not do that though.

Reply:
zoogie i know you've been around long enough to remember when the guide didn't use torrents. do you remember how often the guide had to be updated with new links because they taken down from the host? it was all the time. torrents get around that. so it is better to make the tool be shared via a magnet link than a file host.

Reply:
@pbanj I see.
I just kinda stumbled on this in my notifications. I'll let you guys decide on how to handle b9sTool.

Reply:
As this is still my guide, i will keep your fork as the version i will link to. Maybe add the other one as an option, but i will not remove it as it is too convenient.